helpers.py

```diff
@@ -25,7 +26,8 @@ def public_charge(charge: Charges):
         "custom_css": charge.custom_css,
     }
 
-    if charge.paid:
+    if charge.paid or charge.zeroconf:
+        c["webhook"] = charge.webhook
```

Comment:
- why expose the `webhook`?, this can be a security leak.
- `public_charge` function creates a charge that is safe to expose to the public

Reply:
it was not showing up... it's not passed to view.py